.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,6 +28,9 @@
       - [URL to each sample source, if applicable]
     - Sample license(s):
   - [ ] I have included a syntax highlighting grammar: [URL to grammar repo]
+      <!-- Setting a color is only needed for 'programming' and 'markup' languages and will default to `#cccccc` if not specified. -->
+  - [ ] I have added a color: [Enter color in hex as `#123456` so the color can be seen in the PR body]
```

Comment:
This is a nice-to-have, but it shouldn't be necessary, IMHO. New users are less likely to bother with submissions if there's an excess of instructions they have to follow.